### DISCUSSION TITLE
双线双拨后怎么设置ipv6的负载均衡？？？

### DISCUSSION THREAD

**Discussion by daghy** [coolsnowwolf/lede#9665] [R68S / 电犀牛专区]
![QQ截图20220629124804](https://user-images.githubusercontent.com/34054959/176354664-f7488e09-382a-43b2-af2d-536eb3c99279.jpg)
我设置了2个WAN口进行拨号。分别是WAN跟WAN6。
然后设置2个专门的DHCPV6的接口分别是WAN2跟WAN3。WAN2用接口设置@WAN。WAN3用接口设置@WAN6
这设置时网上找的，具体不懂。
下边是我负载均衡的设置
![QQ截图20220629124837](https://user-images.githubusercontent.com/34054959/176355135-2f66c500-22eb-4935-a75a-7ee19e7b7c51.jpg)
![QQ截图20220629124857](https://user-images.githubusercontent.com/34054959/176355174-b197268e-1e08-4bcb-8596-fe65bfc07fe3.jpg)
![QQ截图20220629124933](https://user-images.githubusercontent.com/34054959/176355239-490b3232-5b5c-4c23-85a4-b25171e97e1c.jpg)
![QQ截图20220629124953](https://user-images.githubusercontent.com/34054959/176355845-49e08539-4fed-4972-9db0-c44b31978866.jpg)

我这么弄完后仅仅只有WAN2口可以连通。看详情是提示WAN3口没有ping通。但我WAN2跟WAN3口的设置时一样的。我2个拨号口也能看到ipv6的地址。我试过把WAN2口暂停，WAN3口在负载均衡状态中又自己连接上了。他们2个口应该能同时在线的。是我哪里还没设置到位吗？？？？

![QQ截图20220629125005](https://user-images.githubusercontent.com/34054959/176356024-7890a0cb-0480-43e0-931b-e4c5da28a9d2.jpg)
![QQ截图20220629125046](https://user-images.githubusercontent.com/34054959/176356385-6c80ffa1-472f-401a-84ea-905029c440c6.jpg)
WAN2口跟WAN3口的network设置
config interface 'wan2'
	option proto 'dhcpv6'
	option reqaddress 'try'
	option reqprefix 'auto'
	option _orig_ifname 'eth1'
	option _orig_bridge 'false'
	option ifname '@wan'
	option metric '50'

config interface 'wan3'
	option proto 'dhcpv6'
	option reqaddress 'try'
	option reqprefix 'auto'
	option _orig_ifname 'eth0'
	option _orig_bridge 'false'
	option ifname '@wan6'
	option metric '70'

WAN2口跟WAN3口的mwan3设置
config policy 'balanced'
	option last_resort 'unreachable'
	list use_member 'wan_m1_w3'
	list use_member 'wan6_m1_w3'
	list use_member 'wan2_m1_w3'
	list use_member 'wan3_m1_w3'

config interface 'wan2'
	option enabled '1'
	option initial_state 'online'
	option family 'ipv6'
	option track_method 'ping'
	option count '1'
	option size '56'
	option check_quality '0'
	option timeout '2'
	option interval '5'
	option failure_interval '5'
	option recovery_interval '5'
	option down '3'
	option up '3'
	option flush_conntrack 'never'
	option reliability '1'
	list track_ip '2001:4860:4860::8888'
	list track_ip '2001:4860:4860::8844'
	list track_ip '240C::6666'
	list track_ip '240C::6644'
	list track_ip '2400:da00::6666'
	list track_ip '2400:3200::1'

config member 'wan2_m1_w3'
	option interface 'wan2'
	option metric '1'
	option weight '1'

config interface 'wan3'
	option enabled '1'
	option initial_state 'online'
	option family 'ipv6'
	option track_method 'ping'
	option count '1'
	option size '56'
	option check_quality '0'
	option timeout '2'
	option interval '5'
	option failure_interval '5'
	option recovery_interval '5'
	option down '3'
	option up '3'
	option flush_conntrack 'never'
	option reliability '1'
	list track_ip '2001:4860:4860::8888'
	list track_ip '2001:4860:4860::8844'
	list track_ip '240C::6666'
	list track_ip '240C::6644'
	list track_ip '2400:da00::6666'
	list track_ip '2400:3200::1'

config member 'wan3_m1_w3'
	option interface 'wan3'
	option metric '1'
	option weight '1'

**Comment by sfrwy** [comment]
总感觉 这个插件有问题 7.19及以后 重启就不会自动启动
